docs/en/developer.md

```diff
@@ -182,3 +183,24 @@ class TaxonomyDirectory_Controller extends Page_Controller
 
 }
 ```
+
+### Using the provided default controller implementation
+If you want to use the provided directory implementation named `TaxonomyDirectoryController`, more or less based on the previous example, the only thing
+you need to do is enable access to the controller functions using a custom `routes.yml` file
+
+```yaml
+
+---
+Name: directoryroutes
+After: framework/routes#coreroutes
+---
+Director:
+  rules:
+    'code/$ID!': 'TaxonomyDirectoryController'
+    
+```
+
+In this example, any request made to `/code/<TAG>` would render a page which contains a list of pages that uses the
+Taxonomy Term specified by `<TAG>`. You can override the provided template `TaxonomyDirectory.ss` in your own theme.
+
+Note: the default directory implementation assumes that you've setup the reverse relation specified by `BasePage`.  
```

Comment:
Add example implementation.

Reply:
I've added an example template to the documentation